src/components/Catalogue/catalogue.tsx

```diff
@@ -12,38 +12,54 @@ import FilterExpand from '../icons/filter_expand';
 import SelFilterExpand from '../icons/sel_filter_expand';
 import { FilterCategory } from '../FilterCategory/filter_category';
 import { title } from 'process';
-import CatalogueAPI from '../../app/api/Catalogue/catalogue';
+import RelicAPI from '../../app/api/Relic/relic';
 
 interface Photo {
     id: number;
     thumbnailUrl: string;
     title: string;
 }
 
+export interface Filters {
+    historicalPeriods: string[];
+    statuses: string[];
+    techniques: string[];
+    collections: string[];
+    categories: string[];
+    [key: string]: string[];
+}
+
 const PAGE_SIZE = 18;
 
 const Catalogue = () => {
+    const navigate = useNavigate();
+    const location = useLocation();
+
     const [items, setItems] = useState<Photo[]>([]);
     const [notFound, setNotFound] = useState(false);
     const [currentPage, setCurrentPage] = useState(1);
     const [selectedCategory, setSelectedCategory] = useState<string | null>(
         null
     );
-    const [selectedFilterOptions, setSelectedFilterOptions] = useState<
-        string[]
-    >([]);
+    const [selectedFilterOptions, setSelectedFilterOptions] = useState<Filters>({
+        historicalPeriods: [],
+        statuses: [],
+        techniques: [],
+        collections: [],
+        categories: [],
+    });
     const [isFilterModalOpen, setIsFilterModalOpen] = useState(false);
-    const navigate = useNavigate();
-    const location = useLocation();
-    const totalPages = 6; //temporarily hardcoded
+    
+    const totalPages = 9; //temporarily hardcoded
 
     const fetchItems = async (page: number) => {
-        //request is under,this is just to display styling
         try {
-            const response = await axios.get<Photo[]>(
-                `https://jsonplaceholder.typicode.com/photos?_page=${page}&_limit=${PAGE_SIZE}`
-            );
-            // const response = await CatalogueAPI.fetchItems(page, PAGE_SIZE);
+            //uncomment this to see styles
+
+            // const response = await axios.get<Photo[]>(
+            //     `https://jsonplaceholder.typicode.com/photos?_page=${page}&_limit=${PAGE_SIZE}`//request is under,this is just to display styling
+            // );
+            const response = await RelicAPI.filterRelics(page, PAGE_SIZE, selectedFilterOptions); 
```

Comment:
should be
`page - 1` because page enumeration starts from 0
![image](https://github.com/ShaggyBeavers/front_end/assets/25075314/b48504e0-4ba9-42a4-85ec-c8b857f4f1d9)